doc/tutorials/acrn_configuration_tool.rst

```diff
@@ -110,6 +110,9 @@ Additional scenario XML elements:
 
 ``SERIAL_CONSOLE`` (a child node of ``DEBUG_OPTIONS``):
   Specify the host serial device is used for hypervisor debugging.
+  This configuration is valid only if SOS legacy_vuart0 is enabled.
```

Comment:
```suggestion
  This configuration is valid only if Service VM ``legacy_vuart0`` is enabled.
```

Please use "Service VM" instead of "SOS" throughout the document.

Reply:
updated.